OKR.md

```diff
@@ -0,0 +1,52 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q2
+
+> Work in Progress
+
+### The daemon is stable and so core is reliable
+
+- `PX` - OWNER - The uncaught errors and memory usage are under control and a daemon can run for a week non stop.
+- `PX` - VictorBjelkholm - One or more js-ipfs daemons are part of the IPFS infrastructure (Bootstraper nodes).
+- `PX` - VictorBjelkholm - js.ipfs.io exposes an IPFS Gateway provided by a js-ipfs daemon.
+
+### The core can handle vast amounts of data
+
+- `PX` - achingbrain - The MFS is implemented in js-ipfs.
+- `PX` - achingbrain - npm on IPFS is used as a test vector of js-ipfs capabilities.
+- `PX` - OWNER - js-ipfs in the browser can fetch a dataset of at least 100Gb (and create archives.js.ipfs.io).
+- `PX` - OWNER - js-ipfs in Node.js can add and transfer a dataset of at least 1TB.
+- `PX` - jacobheun - A S3 IPFS Repo backend exists and it is easily pluggable on `new IPFS`.
+- `PX` - OWNER - The PIN API is finished and merged.
+
+### The APIs are more ergonomic and developer productivity is off the charts
+
+- `PX` - vmx - Complete the first milestone of the Awesome Endeavour - Flow.
+- `PX` - vmx - Roadmap and Kickstart the second milestone for the Awesome Endeavour - Flow.
+- `PX` - fsdiogo - Make js-ipfs uglify friendly again.
+- `PX` - VictorBjelkholm - Every env (i.e Linux, Windows, Mac OS X, Firefox, Chrome, Safari, Edge) test run is parallelized in CI.
+- `PX` - VictorBjelkholm - CLI, HTTP-API and Core Tests of js-ipfs are parallelized in CI.
+- `PX` - fsdiogo - A repo migration utility exists and migration is a good experience for js-ipfs users.
+
+### libp2p levels up, learns new superpowers and a dedicated team is formed
+
+- `PX` - OWNER - libp2p has primitives to keep connections under control.
```

Comment:
Needs an Owner.

Reply:
@diasdavid, count me in to help with the libp2p improvements. Need some time to ramp up with the code base, but should be quick so I can help with the PR reviews and whatnot for now.

Reply:
I can own this one.